src/Runner.Common/Constants.cs

```diff
@@ -153,6 +153,7 @@ public static class ReturnCode
                 public const int RetryableError = 2;
                 public const int RunnerUpdating = 3;
                 public const int RunOnceRunnerUpdating = 4;
+                public const int SessionConflict = 5;
```

Comment:
make sure you double check all the entry script we have to make sure it handles the new exit code as expected.
Most customer use the `./run.sh` or the service entrypoint `.js`

Reply:
Does `run.sh` get updated?

Reply:
there is a `run-helper.sh` which contains the core logic today and it should get updated on runner restart.